includes/admin-menu.php

```diff
@@ -214,6 +214,27 @@ function bt_custom_toolbar_links( $wp_admin_bar ) {
 		);
 		$wp_admin_bar->add_node( $args );
 
+		if ( NerdPress_Helpers::is_nerdpress() && NerdPress_Helpers::is_relay_server_configured() ) {
```

Comment:
Why not move the if statement inside the one in line 217? That way we only call that static method (NerdPress_Helpers::is_nerdpress() once.